src/renderer/components/VideoPreview/VideoPreviewController.tsx

```diff
@@ -34,6 +34,8 @@ export interface VideoPreviewControllerRef {
 interface Props {
   setTime: (time: number) => void;
   setIsPlaying: (isPlaying: boolean) => void;
+  outputVideoLength: number;
+  setOutputVideoLength: React.Dispatch<React.SetStateAction<number>>;
```

Comment:
The type for this can probably just be `(outputVideoLength: number) => void`

Reply:
It works, dispite setOutputVideoLength is of time `React.Dispatch<React.SetStateAction<number>>` when pass it down from the Project.tsx

Reply:
I updated it here: [1474959](https://github.com/chloebrett/mlvet/pull/333/commits/1474959a58a034dfdea00a870c7e59eb18e83ca3)